packages/ui/src/components/Charts/Venn/VennModal.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Modal } from 'antd';
+
+import CardErrorPlaceholder from '../../../view/v2/GridCard/GridCardErrorPlaceholder';
+import { IDictionary } from '../..//QueryBuilder/types';
+import { QueryDictionaryContext } from '../../QueryBuilder/context';
+
+import VennChart, { DEFAULT_VENN_CHART_DICTIONARY, TVennChart, TVennChartDictionary } from '.';
+
+const DEFAULT_VENN_MODAL_DICTIONARY = {
+    ...DEFAULT_VENN_CHART_DICTIONARY,
+    ok: 'Close',
+    title: 'Set operations',
+};
+
+type VennModalDictionary = TVennChartDictionary & {
+    title: string;
+    ok: string;
+};
+
+type VennModalProps = TVennChart & {
+    queryPilldictionary?: IDictionary;
```

Comment:
Un D majuscule je pense non ?
```suggestion
    queryPillDictionary?: IDictionary;
```